tests/test_sentry.py

```diff
@@ -48,3 +53,14 @@ def test_getitem(self, default_env, upload_scripts, wait_for_status):
 
         self.assertEqual(sentry['meteor/0'], sentry.unit['meteor/0'])
         self.assertEqual(sentry['meteor'], list(sentry.unit.values()))
+
+    @patch.object(Talisman, 'wait_for_status')
+    @patch.object(UnitSentry, 'upload_scripts')
+    @patch('amulet.sentry.helpers.default_environment')
+    def test_subordinate(self, default_env, upload_scripts, wait_for_status):
+        default_env.return_value = 'local'
+        wait_for_status.return_value = mock_status
+
+        sentry = Talisman(['meteor', 'rsyslog-forwarder'])
+
+        self.assertEqual(sentry['rsyslog-forwarder/0'], sentry.unit['rsyslog-forwarder/0'])
```

Comment:
I think `self.assertTrue('rsyslog-forwarder/0' in sentry.unit)` would make the intent of the test more clear.